docs/data/base/getting-started/customization/StylingSlots.tsx

```diff
@@ -46,7 +46,7 @@ const Thumb = styled('span')`
   }
 `;
 
-const Input = styled('input')`
+const SwitchInput = styled('input')`
```

Comment:
For consistency with the other demos. But I guess it's also simpler to assume a global namespace. I hope that we have the static CSS extraction, we will be able to use this name to help debug.